Phase1/DesignDocument.md

```diff
@@ -8,16 +8,9 @@ If you found that something in your design wasn't good, tell us about that too!
 Pretending part of your design is good — when you know it isn't — can potentially hurt your mark significantly!
 Acknowledging bad design can earn you marks and demonstrates understanding — especially if you discuss how you could fix it if you had more time!
 
-SINGLE REPONSIBILITY PRINCIPLE: 
-
-OPEN/CLOSED PRINCIPLE: 
-
-LISKOV SUBSTITUTION PRINCIPLE: 
-
-INTEGRATION SEGREGATION PRINCIPLE: 
-
-DEPENDENCY INVERSION PRINCIPLE: 
-
+- Abstract class Level and its subclasses (InterviewLevel, MonthLevel, ReportLevel) adhere to the Open/Closed Principle. InterviewLevel, MonthLevel and ReportLevel all extend Level. GameManager has an instance attribute that stores the current level, and its methods will work regardless of whether its an InterviewLevel, MonthLevel or a ReportLevel. If we wanted to add a new level to our game, we would be able to do so easily by creating a new subclass that extends Level.
```

Comment:
Minor issue, but we could add the headers back instead of deleting them completely..? Or make the principle's you've mentioned bold and standout: easier for others to read.